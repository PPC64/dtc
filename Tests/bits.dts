/dts-v1/;

/ {
	foo8 = /bits/ 8 <(1 + 2 * 3 + 4 * 5 * 6) (1+ 2*3) (3 * 2 + 1)>;
	lt   = <(1<2) (1 < 2) > ;
	le   = <(1<=2) (1 <= 2 )>;
	cond = <(1 * 1 * 2 ? 1 : 2) (0 ? 1 : 2)>;
	paren = <((1+2*3)*3)>;
	foo16 = /bits/ 16 <(1<2) 2 3>;
	foo32 = /bits/ 32 <1 2 3>;
	foo64 = /bits/ 8 <(+1) (-2) 3>;
	foo64 = /bits/ 64 <0xffffffffffffffff>;
	foo322 = /bits/ 32 <0xffffffffffffffff>;
};
