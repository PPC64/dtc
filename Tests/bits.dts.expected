/dts-v1/;

/  {

	foo8 = [7f 07 07];
	lt = <0x1 0x1>;
	le = <0x1 0x1>;
	cond = <0x1 0x2>;
	paren = <0x15>;
	foo16 = [00 01 00 02 00 03];
	foo32 = <0x1 0x2 0x3>;
	foo64 = [01 fe 03];
	foo64 = <0xffffffff 0xffffffff>;
	foo322 = <0xffffffff>;
};
